intl/msg_hash_us.h

```diff
@@ -8364,6 +8364,10 @@ MSG_HASH(
    MENU_ENUM_LABEL_VALUE_LANG_ENGLISH,
    "English"
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_VALUE_LANG_BRITISH_ENGLISH,
+   "British English"
```

Comment:
Since we are using the structure "Language (Region)" for other language choices, shouldn't this be "English (United Kingdom)"?

Reply:
Good point. I'll fix that.

Reply:
done